e2e/plugins/featureFlags.js

```diff
@@ -0,0 +1,95 @@
+const { recorder, event } = require('codeceptjs');
+const Container = require('codeceptjs/lib/container');
+
+const defaultConfig = {
+};
+
+const supportedHelpers = ['Playwright'];
+
+/**
+ * This plugin will listen for setting feature flags and apply them at the moment of page loading.
+ * In this case set feature flags will affect the whole code including  models initialization,
+ * and other similar parts that will run on the the scripts load.
+ */
+
+module.exports = function(config) {
+  const helpers = Container.helpers();
+  let helper;
+
+  for (const helperName of supportedHelpers) {
+    if (Object.keys(helpers).indexOf(helperName) > -1) {
+      helper = helpers[helperName];
+    }
+  }
+
+  if (!helper) {
+    console.error('Feature flags is only supported in Playwright');

```

Comment:
there is `supportedHelpers`, so it would be better to use it here with `join(', ')`